src/travel-details-screens/TripDetailsScreenComponent.tsx

```diff
@@ -71,7 +72,7 @@ export const TripDetailsScreenComponent = ({
   const tripPatternLegs = tripPattern?.legs.map((leg) => {
     return {
       ...leg,
-      mode: !!leg.bookingArrangements ? 'flex' : leg.mode,
+      mode: (!!leg.bookingArrangements ? 'flex' : leg.mode) as AnyMode,
```

Comment:
Casting should generally be avoided if possible. Is it instead possible to write something like `mode: AnyMode = (!!leg.bookingArrangements ? 'flex' : leg.mode)` on its own line?

Reply:
Fixed 👍🏽